build.rs

```diff
@@ -485,24 +469,25 @@ print(json.dumps({
 }))
 "#;
     let json = run_python_script(interpreter, script)?;
-    serde_json::from_str(&json).map_err(|e| format!("Deserializing failed: {}", e))
+    Ok(serde_json::from_str(&json)?)
```

Comment:
removal of these custom error messages is always a gamble: is the error message still as good afterwards?

Reply:
Thanks, changed to use a more specific error message here.